main.py

```
@@ -0,0 +1,185 @@
+        print(f"❌ WebUI起動失敗: {e}")
+        sys.exit(1)
+
+def run_fastapi_ui(args):
+    """FastAPI UI起動"""
+    try:
+        import subprocess
+        
+        # ポート設定
+        port = args.port or 8000
+        
+        print(f"🌐 新しいAIエージェントWebUIを起動中...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)